changelog.md

```diff
@@ -87,7 +87,8 @@
   has been added which can be used to check if a `DateTime` has been initialized.
 
 ## Language changes
-- In newruntime it is now allowed to assign discriminator field without restrictions as long as case object doesn't have custom destructor. Discriminator value doesn't have to be a constant either. If you have custom destructor for case object and you do want to freely assign discriminator fields, it is recommended to refactor object into 2 objects like this:
+- In the newruntime it is now allowed to assign discriminator field without restrictions as long as case object doesn't have custom destructor. Discriminator value doesn't have to be a constant either. If you have custom destructor for case object and you do want to freely assign discriminator fields, it is recommended to refactor object into 2 objects like this:
```

Comment:
`it is now allowed .. as long as`
=> this doesn't specify whether you'll get a CT error if you violate `as long as`